development.ini

```diff
@@ -9,9 +9,11 @@ use = egg:jujugui
 pyramid.reload_templates = true
 pyramid.default_locale_name = en
 
-jujugui.sandbox = true
+jujugui.sandbox = tre
 jujugui.raw = true
 jujugui.combine = true
+jujugui.cachebuster = dev
```

Comment:
We actually want it to use VERSION in prod, so that when we update the gui it breaks the browser cache and the browser gets new files.